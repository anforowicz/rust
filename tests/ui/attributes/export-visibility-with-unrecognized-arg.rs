// This test verfies that `#[export_visibility = ...]` will report an error
// when the argument cannot be parsed as a `SymbolVisibility`.
#![feature(export_visibility)]
#[export_visibility = "unrecognized spec"]
pub static TESTED_STATIC: [u8; 6] = *b"foobar";

fn main() {}
