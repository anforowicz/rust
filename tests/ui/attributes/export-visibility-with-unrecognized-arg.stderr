error: invalid export visibility value
  --> $DIR/export-visibility-with-unrecognized-arg.rs:4:23
   |
LL | #[export_visibility = "unrecognized spec"]
   |                       ^^^^^^^^^^^^^^^^^^^

error: aborting due to 1 previous error

